app/components/atoms/socialMediaButtons/socialMediaButton.component.tsx

```diff
@@ -0,0 +1,77 @@
+import {FC} from 'react';
+import {ButtonProps, StyledButton} from './socialMediaButton.styles';
+
+export const Button: FC<ButtonProps> = ({socialMediaIcon, ...props}) => {
+  let icon;
+  switch (socialMediaIcon) {
+    case 'Instagram':
+      icon = (
+        <a href="https://www.instagram.com/cultoperrocafe/" target="_blank">
```

Comment:
can we just import svg files instead of pasting the entire SVG in the component?